src/en/space-station-14/round-flow/proposals/time-agent.md

```diff
@@ -0,0 +1,83 @@
+* **Miscellaneous**: The future self uses a flash or some other crowd control item - yet again, chosen at random - on the targeted entity. If there are no visible entities on the targeted tile, the future self will target the closest visible person to the tile in a 7 tile radius that isn't the present self. If that still yields no valid results, the future self guards the targeted location for 7 seconds. If, within 7 seconds, a valid entity is seen by it, the future self runs to them while trying to use the crowd control item they currently have. If a wall was targeted, the future self uses something to destroy it. If a disposal unit is targeted, the future self attempts to turn its power off.
+* **Benevolent**: The future self uses 1-2 brute-healing items on the targeted entity if the entity has higher brute damage than burn damage, 1-2 burn-healing items if burn damage is higher than brute damage, or uses the equivalent of Help Intent if the target has no burn damage and no brute damage. If there are no entities on the targeted tile, the future self will target the closest person to the tile in a 7 tile radius that isn't the present self. If that still yields no valid results, the future self guards the targeted location for 7 seconds. If, within 7 seconds, a valid entity is seen by it, the future self runs to them to do the things listed above. If a wall/window was targeted, the future self attempts to fix it with the appropriate tools & materials required. If a disposal unit is targeted, the future self attempts to activate it to send anything in it through.
+
+#### Assist the Past:
+Teleports the agent into a projection of the past - a temporary, empty dimension with featureless figures representing any mobs that were nearby at the time - for however many seconds the unclosed loop had lasted, where all performed actions had already happened before. The damage and other effects that the agent's future self experienced when summoned in reality will be dealt during the assistance in the projection. The center of the temporary dimension’s floor glows the most, and is where **Summon Future Assistance** was targeted originally.
+
+#### Temporal Loop Memories:
+The agent recalls the actions they must perform with **Assist the Past** to close the time loops created by **Summon Future Assistance**.
+
+#### United Synchronization:

```

Comment:
no home stuck references allowed

Reply:
Is that actually a rule? Figured a niche non-visual reference would be fine.